pages/docs/reference/inline-classes.md

```diff
@@ -199,7 +199,7 @@ To remove the warning you have to opt in to the usage of this feature by passing
 <div class="sample" markdown="1" theme="idea" mode="groovy" data-lang="groovy">
 
 ```groovy
-compileKotlin {
+tasks.withType(KotlinCompile).configureEach {
```

Comment:
Maybe we should replace this and other occurrences where a language feature is enabled in a task's compiler args with `kotlin.sourceSets.all { languageSettings.enableLanguageFeature("InlineClasses") }`.
@p7nov WDYT?

Reply:
It's a good idea, let's do.